### PR TITLE
LPS-12345 | master

### PR DESCRIPTION
**Pull request by WYZ-Issac**
Test for training.

This is only for master.

@Tim-Cao

### CODE REVIEW

**Comment by WYZ-Issac** [comment]
ci:test:sf

**Comment by WYZ-Issac** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/myfirsttest/MyFirstTest.testcase [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+		if ("${testPortalInstance}" == "true") {
+			PortalInstances.tearDownCP();
+		}
+		else {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			Site.tearDownCP();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/myfirsttest/MyFirstTest.testcase:23]: No need

File: portal-web/test/functional/com/liferay/portalweb/myfirsttest/MySecondTest.testcase [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+		}
+		else {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "test@liferay.com",
+				userLoginFullName = "Test Test");
+
+			Site.tearDownCP();
+
+			JSONUser.tearDownNonAdminUsers();
+		}
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/myfirsttest/MySecondTest.testcase:25]: No need

**Comment by Tim-Cao** [comment]
ci:close

**Pull request closed without merging**